src/rest/CommunityEndpoints.js

```diff
@@ -0,0 +1,47 @@
+import fetch from 'node-fetch'
+import { Contract } from 'ethers'
+
+import authFetch from './authFetch'
+
+const CommunityProduct = require('../../CommunityProduct.json')
+
+export async function joinCommunity(communityAddress, memberAddress, secret = undefined) {
```

Comment:
Backend only allows the user to join who is logged in the session. So `memberAddress` is possibly redundant, it should be taken from the authenticated session if the session was initiated using private key (ethereum challenge response auth).

Reply:
Starting to think about it... why does backend need it either (in the POST body) if it's supposed to only work for the authenticated address? Maybe you SHOULD be able to generate joinRequests for others, too?

Reply:
I guess there's always the spamming problem, but ofc it doesn't cost anything to authenticate with StreamrClient right now, so in practice it's equivalent.